samples/event-sourcing-esdb-simple/src/main/java/io/eventdriven/ecommerce/api/controller/ShoppingCartsController.java

```diff
@@ -0,0 +1,176 @@
+package io.eventdriven.ecommerce.api.controller;
+
+import io.eventdriven.ecommerce.api.requests.ShoppingCartsRequests;
+import io.eventdriven.ecommerce.core.commands.CommandHandler;
+import io.eventdriven.ecommerce.core.http.ETag;
+import io.eventdriven.ecommerce.core.queries.QueryHandler;
+import io.eventdriven.ecommerce.shoppingcarts.addingproductitem.AddProductItemToShoppingCart;
+import io.eventdriven.ecommerce.shoppingcarts.canceling.CancelShoppingCart;
+import io.eventdriven.ecommerce.shoppingcarts.confirming.ConfirmShoppingCart;
+import io.eventdriven.ecommerce.shoppingcarts.gettingbyid.GetShoppingCartById;
+import io.eventdriven.ecommerce.shoppingcarts.gettingbyid.ShoppingCartDetails;
+import io.eventdriven.ecommerce.shoppingcarts.gettingcarts.GetShoppingCarts;
+import io.eventdriven.ecommerce.shoppingcarts.gettingcarts.ShoppingCartShortInfo;
+import io.eventdriven.ecommerce.shoppingcarts.opening.OpenShoppingCart;
+import io.eventdriven.ecommerce.shoppingcarts.productitems.PricedProductItem;
+import io.eventdriven.ecommerce.shoppingcarts.productitems.ProductItem;
+import io.eventdriven.ecommerce.shoppingcarts.removingproductitem.RemoveProductItemFromShoppingCart;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.enums.ParameterIn;
+import io.swagger.v3.oas.annotations.media.Schema;
+import org.springframework.data.domain.Page;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+
+@RestController
+@RequestMapping("/shopping-carts")
+public class ShoppingCartsController {
```

Comment:
```suggestion
class ShoppingCartsController {
```

Reply:
I also did more work around scopes here: https://github.com/oskardudycz/EventSourcing.JVM/pull/1/commits/6019244a323d2cbc5d752555435fa77e705b97cd. Yet, it seems that I need to work on it a bit more, as package scope is not the same as `internal` in C#, and it might be that I need to structure a bit more my classes layout. In C# it's applied for the `assembly` so the main package, not only to the specific one as in Java.